code/game/objects/items/robot/robot_items.dm

```diff
@@ -336,15 +336,20 @@
 		cooldown = world.time + 600
 		log_game("[user.ckey]([user]) used an emagged Cyborg Harm Alarm in ([user.x],[user.y],[user.z])")
 
+#define DISPENSE_LOLLIPOP_MODE "dispense_lollipop"
+#define THROW_LOLLIPOP_MODE "throw_lollipop"
+#define THROW_GUMBALL_MODE "throw_gumball"

```

Comment:
Use numbers instead of strings

Reply:
Why though

Reply:
The candyfab does get spammed from time to time. Maybe the change from string tests to integer tests is worth it.

Reply:
It's standard to use numbers for non user facing enums wherever possible

Reply:
make the change please